### PR TITLE
Demo 2418 add summary for dashboards

### PR DESCRIPTION
**Pull request by gabeobrien**
This PR add support for displaying a summary of the newly added 'global' instrumentors.

### CODE REVIEW

File: tests/install/install_provider_builder.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require "json"
+require "mocha/mock"
+require "./src/isntall/orchestrator"
+
+module Tests
+  module Services
+    class InstallProviderBuilder
```
Comment (Julien4218) [inline tests/install/install_provider_builder.rb:7]: Is this file obsolete?

File: src/summary/orchestrator.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Orchestrator
 
     def initialize(context)
       @context = context
-      @summary_composer = Summary::Composer.new()
+      @summary_composer = Summary::Composer.new(context)
```
Comment (Julien4218) [inline src/summary/orchestrator.rb:11]: This doesnt seem to match the constructor definition on SummaryComposer?

File: tests/install/install_provider_builder.rb (hunk 1 above)
Comment (gabeobrien) [reply tests/install/install_provider_builder.rb]: yes removed

File: tests/provision/provision_provider_builder.rb [hunk 3]
```diff
@@ -20,6 +20,8 @@ def service_host(id, ip)
         output_param["id"] = id
         output_param["ip"] = ip
         @output_params.push(output_param)
+
+        return @parent_builder
```
Comment (aswanson-nr) [inline tests/provision/provision_provider_builder.rb:24]: Is this a change we want to keep even though we didn't use it?

Comment (gabeobrien) [reply tests/provision/provision_provider_builder.rb:24]: Good catch. I think we can take that out because non of our changes need this to be done.

**Review by Julien4218 — APPROVED** [review]
This looks good, will test.

**Pull request merged**